config/application.rb

```diff
@@ -18,5 +18,6 @@ class Application < Rails::Application
     #
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
+    PASSWORD = "password123"
```

Comment:
🚫 **[Bearer]** <[ruby_lang_hardcoded_secret](https://docs.bearer.com/reference/rules/ruby_lang_hardcoded_secret)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>
# Hard-coded secret detected.
## Description

Applications should store secret values securely and not as literal values
in the source code.

## Remediations

✅ Retrieve secrets from a secure location at runtime

## Resources
- [OWASP hardcoded passwords](https://owasp.org/www-community/vulnerabilities/Use_of_hard-coded_password)
- [OWASP secrets management cheat sheet](https://cheatsheetseries.owasp.org/cheatsheets/Secrets_Management_Cheat_Sheet.html#21-high-availability)